source/includes/fundamentals/code-snippets/libmongocrypt-gradle-versioned.rst

```diff
@@ -0,0 +1,5 @@
+.. code-block:: sh
```

Comment:
Nitpick: You could use "groovy" instead of "sh" here, similar to this
https://raw.githubusercontent.com/mongodb/docs-java/master/source/includes/quick-start/gradle-versioned.rst